.eslintrc.js

```diff
@@ -2,5 +2,8 @@ module.exports = {
     extends: ["hive/vue", "hive/prettier"],
     rules: {
         "vue/no-v-html": ["off"]
+    },
+    env: {
+        jest: true

```

Comment:
no need for jest

Reply:
Didn't we mention jest specifically in the tech in review? Why mocha instead?

Reply:
<img width="1168" alt="Screenshot 2020-05-11 at 11 34 23" src="https://user-images.githubusercontent.com/25441/81552439-6603b180-937b-11ea-87b5-8b5cb59796c6.png">

Reply:
If it works with mocha we're not going to change